polars/polars-core/src/datatypes/any_value.rs

```diff
@@ -71,8 +71,10 @@ pub enum AnyValue<'a> {
     // If syncptr is_null the data is in the rev-map
     // otherwise it is in the array pointer
     Categorical(u32, &'a RevMapping, SyncPtr<Utf8Array<i64>>),
-    /// Nested type, contains arrays that are filled with one of the datetypes.
+    /// Nested type, contains arrays that are filled with one of the datatypes.
     List(Series),
+    #[cfg(feature = "dtype-fixed-size-list")]
+    FixedSizeList(Series, usize),
```

Comment:
I don't know exactly how this `AnyValue` works, but based on `List(Series)` I'm guessing this is right